### PR TITLE
Contact Section

### PR DESCRIPTION
**Pull request by KhaledHN**

### CODE REVIEW

File: components/Contact.jsx [hunk 1]
```diff
@@ -20,18 +31,20 @@ const Contact = () => {
           <div className="col-span-3 lg:col-span-2 w-full h-full shadow-xl shadow-gray-400 rounded-xl p-4 ">
             <div className=" lg:p-3 h-full ">
               <div>
-                <img
+                <Image
                   className=" rounded-xl hover:scale-105 ease-in duration-300 "
-                  src="contact assets/contactpic.jpg"
+                  src="/contact assets/contactpic.jpg"
```
Comment (HazemKhaled) [inline components/Contact.jsx]: fix contact us assets directory name, should be into the assets folder no spaces

**Review by HazemKhaled — CHANGES_REQUESTED** [review]
Perfect, small fix

**Review by HazemKhaled — CHANGES_REQUESTED** [review]
Keep using one package manager

File: components/Contact.jsx (hunk 1 above)
Comment (HazemKhaled) [reply components/Contact.jsx]: any update?

**Review by HazemKhaled — APPROVED** [review]

File: components/Contact.jsx (hunk 1 above)
Comment (KhaledHN) [reply components/Contact.jsx]: Done

**Pull request merged**